examples/chat/src/chatScreen.tsx

```diff
@@ -61,7 +64,10 @@ export function ChatScreen(props: { chatID: string }) {
           chat
             .slice(-showNLastMessages)
             .reverse() // this plus flex-col-reverse on ChatBody gives us scroll-to-bottom behavior
```

Comment:
Might it be helpful here to add a bit to the comment clarifying that we can only reverse because we already sliced and so are not working on the value directly to avoid confusion, or is that too verbose? I guess TS would warn if someone tries to mutate, so maybe it’s overkill?